crates/vault-registry/src/lib.rs

```diff
@@ -571,68 +589,54 @@ impl<T: Config> Pallet<T> {
         LiquidationVault::<T>::get()
     }
 
-    /// Locks an `amount` of collateral to be used for backing tokens
+    /// Deposit an `amount` of collateral to be used for backing tokens
     ///
     /// # Arguments
-    /// * `vault_id` - the id of the vault from which to increase to-be-issued tokens
-    /// * `amount` - the amount of collateral to be locked
+    /// * `vault_id` - the id of the vault
+    /// * `amount` - the amount of collateral
     pub fn try_deposit_collateral(vault_id: &T::AccountId, amount: Backing<T>) -> DispatchResult {
         let mut vault = Self::get_active_rich_vault_from_id(vault_id)?;
 
         // will fail if free_balance is insufficient
         ext::collateral::lock::<T>(vault_id, amount)?;
-        Pallet::<T>::increase_total_backing_collateral(amount)?;
-
-        vault.increase_backing_collateral(amount)?;
+        Self::increase_total_backing_collateral(amount)?;
+        vault.try_deposit_collateral(amount)?;
 
         Ok(())
     }
 
+    /// Withdraw an `amount` of collateral without checking collateralization
+    ///
+    /// # Arguments
+    /// * `vault_id` - the id of the vault
+    /// * `amount` - the amount of collateral
     pub fn force_withdraw_collateral(vault_id: &T::AccountId, amount: Backing<T>) -> DispatchResult {
-        Self::force_withdraw_collateral_to_address(vault_id, amount, vault_id)
-    }
-
-    pub fn force_withdraw_collateral_to_address(
-        vault_id: &T::AccountId,
-        amount: Backing<T>,
-        payee_id: &T::AccountId,
-    ) -> DispatchResult {
         let mut vault = Self::get_rich_vault_from_id(vault_id)?;
-        if !vault_id.eq(payee_id) {
-            Self::slash_collateral(
-                CurrencySource::Backing(vault_id.clone()),
-                CurrencySource::FreeBalance(payee_id.clone()),
-                amount,
-            )?;
-        } else {
-            ext::collateral::release_collateral::<T>(vault_id, amount)?;
-            vault.decrease_backing_collateral(amount)?;
-        }
-        Pallet::<T>::decrease_total_backing_collateral(amount)?;
+
+        // will fail if reserved_balance is insufficient
+        ext::collateral::release_collateral::<T>(vault_id, amount)?;
+        vault.try_withdraw_collateral(amount)?;
```

Comment:
it's a little confusing to have the call stack be:
pub function: `try_withdraw_collateral` -> calls `force_withdraw_collateral` -> calls `vault.try_withdraw_callateral`